nix/desktop/macos/default.nix

```
@@ -1,15 +1,14 @@
-{ stdenv, callPackage, pkgs,
-  darwin, qt5, status-go, baseImageFactory }:
+{ stdenv, lib, callPackage, mkShell,
+  gnupg22, darwin, qt5, status-go, baseImageFactory }:
 
 with darwin.apple_sdk.frameworks;
 
 assert stdenv.isDarwin;
-
 let
-  inherit (stdenv.lib) concatStrings catAttrs;
+  inherit (lib) concatStrings catAttrs;
   baseImage = baseImageFactory "macos";
 
-in {
+in rec {
   buildInputs = [
     pkgs.gnupg22
```

I think you forgot to remove `pkgs.` here:
```suggestion
    gnupg22
```

---

true